src/database.clj

```diff
@@ -0,0 +1,63 @@
+(ns database
+  (:require [clojure.java.jdbc :as j]))
+
+(def db {:dbtype "postgresql"
+         :dbname "ttt"
+         :host "localhost"
+         :user "angie"
+         :password "MtNmWUsC"})
+
+(def stats-sql (j/create-table-ddl :stats [[:stats_id :serial "PRIMARY KEY"]
+                                           [:username :text "UNIQUE"]
+                                           [:wins :int "DEFAULT 0"]
+                                           [:ties :int "DEFAULT 0"]
+                                           [:losses :int "DEFAULT 0"]]
+                                   {:conditional? true}))
+
+
+(j/execute! db [stats-sql])
+
+(defn- retrieve-username
```

Comment:
would it be more accurate to name it as something like `retrieve-stats-by-username`?